packages/flutter_tools/test/commands.shard/hermetic/proxy_validator_test.dart

```diff
@@ -2,13 +2,41 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io' as io;
+import 'dart:typed_data';
+
+import 'package:flutter_tools/src/base/io.dart';
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/doctor_validator.dart';
 import 'package:flutter_tools/src/proxy_validator.dart';
 
 import '../../src/common.dart';
 
 void main() {
+  setUp(() {
+    setNetworkInterfaceLister(
+      ({
+        bool includeLoopback = true,
+        bool includeLinkLocal = true,
+        InternetAddressType type = InternetAddressType.any,
+      }) async {
+        final List<FakeNetworkInterface> interfaces = <FakeNetworkInterface>[
+          FakeNetworkInterface(<FakeInternetAddress>[
+            const FakeInternetAddress('127.0.0.1')
+          ]),
+          FakeNetworkInterface(<FakeInternetAddress>[
+            const FakeInternetAddress('::1')
+          ])
+        ];
+
+        return Future<List<NetworkInterface>>.value(interfaces);
+    });
+  });
+
+  tearDown(() {
+    resetNetworkInterfaceLister();
+  });
+  
```

Comment:
The presubmit check is complaining about a trailing whitespace character on this line.